index.js

```diff
@@ -0,0 +1,23 @@
+//*  2020 - 03 - 19 //
+// 03 - 19 - 2020 //
+
+let oldDate = "2020 - 03 - 19";
+
+const date = oldDate.split(" ");
+
+let temp = date[0];
+date[0] = date[4];
+date[4] = temp;
+
+let temp2 = date [0];
```

Comment:
Suggestion: Seems your are reusing the swapping logic twice. Would be a good idea to encapsulate that into a function for reusability.